source/develop/release-management/features/network/lldp.html.md

```diff
@@ -0,0 +1,462 @@
+    </properties>
+    <type>7</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Management Address</name>
+    <properties>
+      <property>
+        <name>management address subtype</name>
+        <value>AFI 802</value>
+      </property>
+      <property>
+        <name>management address</name>
+        <value>00:01:30:f9:ad:a0</value>
+      </property>
+    </properties>
+    <type>7</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Port VLAN Id</name>
+    <oui>32962</oui>
+    <properties>
+      <property>
+        <name>port vlan id</name>
+        <value>488</value>
+      </property>
+    </properties>
+    <subtype>1</subtype>
+    <type>127</type>
+  </link_layer_discovery_protocol_element>
+  <link_layer_discovery_protocol_element>
+    <name>Port VLAN Id</name>
+    <oui>32962</oui>
+    <properties>
+      <property>
+        <name>vlan id</name>
+        <value>488</value>
+      </property>
+      <property>
+        <name>vlan name</name>
+        <value>v2-0488-03-0505</value>
+      </property>
+    </properties>
+    <subtype>3</subtype>
+    <type>127</type>
+  </link_layer_discovery_protocol_element>
+</link_layer_discovery_protocol_elements>
+```
+
+### Graphical User Interface
+
+Graphical User Interface is a [folow-up feature](#follow-up-features)
+
+## Installation/Upgrade
+
+The VDSM package would require the package lldpad and ask the service lldpad to
+start.
+
+Describe how the feature will affect new or existing installations.
+
+## User work-flows
+
+The user gets the information about the networking neighbor via REST-API.
+
+## Dependencies
+
+### oVirt Engine Side
+
+No additional dependencies on oVirt Engine side.
+
+### VDSM Host Side
+
+The package lldpad has to be installed and the demon lldpad has to be running.
+
+## Documentation & External references
+
+[IEEE Std 802.1AB™-2016, IEEE Standard for Local and metropolitan area networks—Station and Media Access Control Connectivity Discovery][1]
+
+[1]: http://standards.ieee.org/getieee802/download/802.1AB-2016.zip
+
+[Linux Ethernet Bonding Driver HOWTO][2]
+
+[2]: https://www.kernel.org/doc/Documentation/networking/bonding.txt
+
+[Ansible Docs » lldp - get details reported by lldp][3]
+
+[3]: http://docs.ansible.com/ansible/lldp_module.html
+
+[OpenStack: Introspection LLDP reporting][4]
+
+[4]: https://specs.openstack.org/openstack/ironic-inspector-specs/specs/lldp-reporting.html
+
+[SampleCaptures from Wireshark's wiki][5]
+
+[5]: https://wiki.wireshark.org/SampleCaptures?action=AttachFile&do=view&target=lldp.detailed.pcap
+
+## Testing
+
+Ensure [prerequisites](#prerequisites) and use REST-API like in
+[example](#rest-api) above.
+
+## Contingency Plan
+
+Explain what will be done in the event that the feature is not ready in time.
+
+## Follow-Up Features
+
+Another benefit of this feature is, that the information can be used by oVirt
```

Comment:
More interesting would be 'auto-configuring' hosts in the desired networking config. Assuming that interfaces ARE correctly patched to the correct ports, which have VLAN and such already assigned, it may be able to already configure the networks on the host when adding the host.